test/test_file_processing.py

```diff
@@ -14,7 +38,7 @@ def test_thermal_video(self, helper, file_processing):
         helper.given_a_recording(self)
 
         # Get a recording to process.
-        recording = file_processing.get("thermalRaw", "getMetadata")
+        file_processing.get("thermalRaw", "getMetadata")
         assert recording["processingState"] == "getMetadata"
```

Comment:
Doesn't look like recording is defined anymore?

Reply:
oh yep sorted now